docs/manual/ar/introduction/Useful-links.html

```diff
@@ -26,7 +26,7 @@ <h2>الدروس والدورات</h2>
 		<h3>الشروع في العمل مع three.js</h3>
 		<ul>
 			<li>
-				[link:https://threejsfundamentals.org/threejs/lessons/threejs-fundamentals.html Three.js Fundamentals starting lesson]
+				[link:https://threejs.org/manual/ Three.js Fundamentals starting lesson]
```

Comment:
Please use `https://threejs.org/manual/#en/fundamentals` instead otherwise the user lands to an empty content page.

`https://threejs.org/manual/#en/fundamentals` is also used by the homepage.